chartfx-chart/src/main/java/de/gsi/chart/renderer/spi/LabelledMarkerRenderer.java

```diff
@@ -211,7 +213,7 @@ public boolean isVerticalMarker() {
     }
 
     @Override
-    public void render(final GraphicsContext gc, final Chart chart, final int dataSetOffset,
+    public List<DataSet> render(final GraphicsContext gc, final Chart chart, final int dataSetOffset,
```

Comment:
Codacy found an issue: [The method 'render(GraphicsContext, Chart, int, ObservableList)' has an NPath complexity of 216](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5764587)